process/logsevents/serialize.go

```diff
@@ -92,3 +93,30 @@ func (logsAndEventsProcessor) SerializeTokens(tokens []*data.TokenInfo) ([]*byte
 
 	return buffSlice.Buffers(), nil
 }
+
+func (lep *logsAndEventsProcessor) SerializeDelegators(delegators map[string]*data.Delegator) ([]*bytes.Buffer, error) {
+	buffSlice := data.NewBufferSlice()
+	for _, delegator := range delegators {
+		id := lep.computeDelegatorID(delegator)
+		meta := []byte(fmt.Sprintf(`{ "index" : { "_id" : "%s" } }%s`, id, "\n"))
+		serializedData, errMarshal := json.Marshal(delegator)
+		if errMarshal != nil {
+			return nil, errMarshal
+		}
+
+		err := buffSlice.PutData(meta, serializedData)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return buffSlice.Buffers(), nil
+}
+
+func (lep *logsAndEventsProcessor) computeDelegatorID(delegator *data.Delegator) string {
+	delegatorContract := delegator.Address + delegator.Contract
+
+	hashBytes := lep.hahser.Compute(delegatorContract)
+
+	return base64.StdEncoding.EncodeToString(hashBytes)
```

Comment:
I don't think this is ok. if we want to search in this index based on an address and a contract, we'd have to convert it to base64 at first. maybe stick to plain string